benchmark/cluster/vgg16/README.md

```diff
@@ -62,6 +62,28 @@ Setting environment variable: `MKL_NUM_THREADS=1`.
 *The performance gap between Fuild and v2 comes from the network interference.*
 
 
+## Test Result (GPU with flowers)
+
+### Hardware Infomation
+
+- GPU: NVIDIA Tesla P40, Driver Version: 375.26
```

Comment:
FYI: we may need to update the Nvidia driver in the cluster soon, the NCCL 2 we currently uses does not support 380.x (tested work on the latest 390.x).